dashboard/README.md

```diff
@@ -42,20 +42,20 @@ Navigate to `dashboard` directory and run following command
 	$ npm install
 	```
 
-	> NOTE: 
-For development and test, create a `.env` file in the root directory (`/dashboard/`) and declare the environment variable `PBENCH_SERVER`.
+## Development and Test 
+
+Create a `.env` file in the root directory (`/dashboard/`) and declare the environment variable `PBENCH_SERVER`.
 This `PBENCH_SERVER` environment variable is the base URL for all API calls and it should point to a real pbench server.
 	```bash
 	PBENCH_SERVER=<pbench server url>
 	```
 
-4. In order to start the development express server and run the application use the following command 
+In order to start the development express server and run the application use the following command 
 	```bash
 	$ npm run dev
 	```
 	> Note: The application runs on http://localhost:3000.
```

Comment:
Now that this is no longer inside a list, it doesn't seem to be rendering properly -- the "fenced code blocks" are ending up in-line instead of as blocks.

It looks like you need to remove the tab characters at the beginnings of the lines (49-51, 54-57).

Reply:
Updated